### DISCUSSION TITLE
New Commits

### DISCUSSION THREAD

**Discussion by sirinath** [vnmakarov/mir#300] [General]
The number of commits seam to be decreasing. Wondering what is the reason behind that.

**Comment by vnmakarov** [comment]
Thank you for expressing your concerns.  I hear you.

I'd like to say that I am not abandoning the project and am not losing interest in MIR-project.  Simply I have too much technical debth and I am not a multi-tasker.

Currently I am actively working on https://github.com/vnmakarov/ruby.  Ruby JIT was a major motivation for me to start MIR-project.  I am getting an experience with using MIR for Ruby JIT and it will be reflected on the future MIR development during next year.  I will be busier since Nov to Apr because I have to work on GCC register allocator issues as the work on the next GCC release will start in Nov. 

The major MIR-development is now on the bbv branch.  GitHub does not report commits on branches differently from master.  Therefore many my commits to MIR and ruby are not reported.

Still if there are some urgent bugs in the current release, I am ready to fix this in timely manner.  Also I'd like to involve more people to participate in MIR development.
 
I hope I answered your question.

**Reply by sirinath** [reply]
This is a relief. I was following the project with much interest.
